### PR TITLE
Add plugin dependencies to setup.py

### PR DESCRIPTION
**Pull request by sbesson**
Also include classifiers and license

### CODE REVIEW

**Comment by sbesson** [comment]
Noticed while working on #28 and #29 and trying to re-add all the dependencies of the plugin. With the last release of `ome-zarr-py` (https://github.com/ome/ome-zarr-py/pull/55), `scikit-image` is also transitively declared as a dependency.

Minimally tested with

```
FROM centos:7

RUN yum install -y git python3 fontconfig
RUN useradd -ms /bin/bash zarr
USER zarr

RUN python3 -m venv /tmp/venv && /tmp/venv/bin/pip install -U pip wheel && /tmp/venv/bin/pip install https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.2.1/zeroc_ice-3.6.5-cp36-cp36m-linux_x86_64.whl
RUN /tmp/venv/bin/pip install git+https://github.com/sbesson/omero-cli-zarr@plugin_dependencies

ENTRYPOINT ["/tmp/venv/bin/omero", "zarr", "-h"]
```

File: setup.py [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+        "Intended Audience :: Information Technology",
+        "License :: OSI Approved :: GNU General Public License v2 " "or later (GPLv2+)",
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 3",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    author="The Open Microscopy Team",
     author_email="",
+    license="",
```
Comment (manics) [inline setup.py:26]: I was going to suggest something here, but ended up in a rabbit hole... https://github.com/pypa/packaging.python.org/issues/95

Comment (sbesson) [reply setup.py:26]: Agreed. https://packaging.python.org/guides/distributing-packages-using-setuptools/#license suggests that the `classifiers` should be the reference place for the project license in all cases. Happy to remove the `license` keyword if the vocabulary if unclear as I could not find a list of official terms either.

Comment (manics) [reply setup.py:26]: https://github.com/ome/omero-py/blob/dab7f1790ec0c0b5f0db0beaf3948b328a41e469/setup.py#L179-L233 doesn't have a `license` field and no-one's said anything.

**Comment by sbesson** [comment]
I think the remaining pre-commit issues are now unrelated to this PR but to changes already merged on `master`

**Comment by joshmoore** [comment]
Travis is gone in the mainline. Merging.

**Pull request merged**